scalafmt-core/shared/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -110,7 +110,9 @@ case class Newlines(
     beforeImplicitKWInVerticalMultiline: Boolean = false,
     alwaysBeforeElseAfterCurlyIf: Boolean = false,
     alwaysBeforeMultilineDef: Boolean = true,
-    avoidAfterYield: Boolean = true
+    avoidAfterYield: Boolean = true,
+    avoidEmptyLineAfterOpenCurly: Boolean = false,
+    avoidEmptyLineBeforeClosingCurly: Boolean = false
```

Comment:
Great contribution, I want it in my `.scalafmt.conf` :smile: 

Is there a use case when this settings will be turned on separatly? May be replace it by one configuration key (something like `avoidEmptyLinesAroundBlock`)?

Reply:
I split it into two settings to make it as flexible as possible, but I'd also always want to have both settings turned on. One configuration key would be fine with me.